guides/introduction/quick_start_guide.md

```diff
@@ -114,3 +114,181 @@ This will lead you to a web page with a search bar that will allow you to search
 <br>
 <br>
 <MovieGif />
+
+## Integrate to Your Project
+
+The only step missing now is adding the search bar to your project. The easiest way of achieving this is to use [instant-meilisearch](https://github.com/meilisearch/instant-meilisearch): A developer tool that generates all the search components needed to start searching.
+
+[Instant-meilisearch](https://github.com/meilisearch/instant-meilisearch) works on common front-end environments as: [Javascript](https://developer.mozilla.org/en-US/docs/Web/JavaScript), [React](https://github.com/meilisearch/meilisearch-react), and [VueJs](https://github.com/meilisearch/meilisearch-vue).
+
+#### Lets Try:
+
+- Create an `html` file, for example, `index.html`.
+- Open it in a text editor (notepad, sublime text, VSC, or other).
+- Copy-paste any of the code examples bellow and save the file.
+- Open `index.html` in your browser (double click on it in your folder).
+
+_We use browser builds for ease of integration. It is possible to do this with `npm` or `yarn`. Please refer to [instant-meilisearch](https://github.com/meilisearch/instant-meilisearch) for documentation._
+
+:::: tabs
+
+::: tab Javascript
+
+The following code sample uses plain [javascript](https://developer.mozilla.org/en-US/docs/Web/JavaScript).
+
+```html
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+  </head>
+  <body>
+    <div>
+      <div id="searchbox" focus></div>
+      <div id="hits"></div>
+    </div>
+
+    <script src="https://cdn.jsdelivr.net/npm/@meilisearch/instant-meilisearch/dist/instant-meilisearch.umd.min.js"></script>
+    <script src="https://cdn.jsdelivr.net/npm/instantsearch.js@4"></script>
+    <script>
+        const search = instantsearch({
+            indexName: "movies",
+            searchClient: instantMeiliSearch(
+                "http://localhost:7700",
+                "masterKey",
```

Comment:
We don't use any key throughout the getting started. Maybe should we remove this one too?